src/Storage/Commands.Storage/help/Get-AzureStorageServiceProperty.md

```diff
@@ -0,0 +1,87 @@
+---
+external help file: Microsoft.WindowsAzure.Commands.Storage.dll-Help.xml
+Module Name: Azure.Storage
+online version: 
+schema: 2.0.0
+---
+
+# Get-AzureStorageServiceProperty
+
+## SYNOPSIS
+Gets properties for Azure Storage services.
+
+## SYNTAX
+
+```
+Get-AzureStorageServiceProperty [-ServiceType] <StorageServiceType> [-Context <IStorageContext>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Get-AzureStorageServiceProperty** cmdlet gets the properties for Azure Storage services.
+
+## EXAMPLES
+
+### Example 1: Get  Azure Storage services property of the Blob service
+```
+C:\PS>Get-AzureStorageServiceProperty -ServiceType Blob
```

Comment:
@blueww would you mind providing the output from this command as well so users have an idea of what will be returned?

Reply:
The output is provided